server/text_generation_server/models/causal_lm.py

```diff
@@ -292,7 +293,7 @@ def from_pb(
         # this means that we cannot shift inputs to the left after a long input sequence
         # was filtered out
         new_bs = round_up(len(requests), PREFILL_BATCH_BUCKET_SIZE)
-        dummy_inputs = ["?"] * (new_bs - len(requests))
+        dummy_inputs = [get_dummy_input(tokenizer)] * (new_bs - len(requests))
```

Comment:
I don't understand why we need to replace "?". Won't it be better to just modify encode logic?